test-clients/src/main/java/com/hedera/services/bdd/spec/assertions/ContractInfoAsserts.java

```diff
@@ -87,40 +93,29 @@ public ContractInfoAsserts propertiesInheritedFrom(String contract) {
 		registerProvider((spec, o) -> {
 			ContractInfo expected = spec.registry().getContractInfo(contract);
 			ContractInfo actual = object2ContractInfo(o);
-			Assert.assertEquals("Bad expiry time!",
+			Assertions.assertEquals(
 					expected.getExpirationTime(),
-					actual.getExpirationTime());
-			Assert.assertEquals("Bad auto renew period!",
+					actual.getExpirationTime(),
+					"Bad expiry time!");
+			Assertions.assertEquals(
 					expected.getAutoRenewPeriod(),
-					actual.getAutoRenewPeriod());
-			Assert.assertEquals("Bad admin key!",
-					expected.getAdminKey(),
-					actual.getAdminKey());
-			Assert.assertEquals("Bad memo!",
+					actual.getAutoRenewPeriod(),
+					"Bad auto renew period!");
+			Assertions.assertEquals(expected.getAdminKey(),
+					actual.getAdminKey(),
+					"Bad admin key!");
+			Assertions.assertEquals(
 					expected.getMemo(),
-					actual.getMemo());
+					actual.getMemo(),
+					"Bad memo!");
 		});
 		return this;
 	}
 
 	public ContractInfoAsserts adminKey(String expectedKeyName) {
 		registerProvider((spec, o) -> {
 			Key expectedKey = spec.registry().getKey(expectedKeyName);
-			Assert.assertEquals("Bad admin key!", expectedKey, object2ContractInfo(o).getAdminKey());
-		});
-		return this;
-	}
-
-	public ContractInfoAsserts nonNullAccountId() {
```

Comment:
Similarly here as well. Good asserts to have although we are not using them in the current EET tests.